features/support/object_model_pages/ivl/iap/ivl_iap_other_questions_page.rb

```diff
@@ -7,6 +7,14 @@ def self.is_pregnant_no_radiobtn
     'is_pregnant_no'
   end
 
+  def self.no_primary_caregiver_radiobtn
+    'input[id="is_primary_caregiver_no"]'
+  end
+
+  def self.no_primary_caregiver_radiobtn_not_checked
+    '#is_primary_caregiver_no:not(:checked)'
```

Comment:
I'm not sure if this is a valuable test @jacobkagon 

we may be testing how rails works, rather than our business logic

Reply:
@markgoho the code change from the applicant model is reflected in the view. What do you think would be a more valuable test?

Reply:
I could make a spec as well 
`context "is primary caregiver" do
    it "should not have a default value" do
      expect(applicant.is_primary_caregiver).to be(nil)
    end
  end`

Reply:
rspecs are going to be faster and better for testing business logic, so I'd definitely support moving this part of the test over there and eliminating the cucumber part

Reply:
I agree, but shouldn't changes to the view also be tested?